MobiusCore/Source/Mobius.swift

```diff
@@ -47,7 +47,7 @@ public extension Mobius {
         return Builder(
             update: update,
             effectHandler: effectHandler,
-            initiator: { First(model: $0) },
+            initiate: nil,
```

Comment:
Not a big deal, but it seems the convention in this file is to stub out things which are not provided rather than making them `nil`.
I think `nil` is the way we should go, but it does look inconsistent within the context of the existing file.

Reply:
As implemented, initiators and initial effects are mutually exclusive (so I don’t have to define an ordering rule). Since closures don’t have identity, this means we have to track whether an explicit initiator was set, and an optional is the cleanest way.